Rhea/AppDelegate.m

```diff
@@ -169,6 +169,11 @@ - (void)userNotificationCenter:(NSUserNotificationCenter *)center didActivateNot
     }
 }
 
+- (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center
+     shouldPresentNotification:(NSUserNotification *)notification {
```

Comment:
Minor nit, but could you use the code formatting I have been using throughout the rest of the project? Would have the method signature all on one line and the`{` on the following line, like

```
- (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center shouldPresentNotification:(NSUserNotification *)notification
{
...
```